src/test/java/com/stripe/functional/StripeResponseTest.java

```diff
@@ -0,0 +1,62 @@
+package com.stripe.functional;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+
+import com.stripe.Stripe;
+import com.stripe.model.Customer;
+import com.stripe.model.CustomerCollection;
+
+import com.stripe.BaseStripeFunctionalTest;
+import com.stripe.exception.APIConnectionException;
+import com.stripe.exception.APIException;
+import com.stripe.exception.AuthenticationException;
+import com.stripe.exception.CardException;
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.StripeResponse;
+import com.stripe.net.RequestOptions;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThat;
+
+import java.util.HashMap;
+
+public class StripeResponseTest extends BaseStripeFunctionalTest {
+  @Test
+	public void testResponseIncluded() throws
+		AuthenticationException,
+		InvalidRequestException,
+		APIException,
+		APIConnectionException,
+		CardException {
+			String idempotencyKey = Long.toString(System.currentTimeMillis());
+			RequestOptions includeResponseRequestOptions = RequestOptions.builder()
```

Comment:
Mind renaming this to just `requestOptions`, as we removed the `includeResponse` flag?